app/src/main/res/values/strings.xml

```diff
@@ -19,27 +19,29 @@
     <string name="url_mail" translatable="false">mailto:dede.hu@qq.com</string>
     <string name="url_sponsor" translatable="false">https://github.com/hushenghao/AndroidEasterEggs/wiki/Sponsor</string>
 
-    <string name="char_separator">/</string>
+    <string name="char_separator" translatable="false">/</string>
     <string name="char_en_dash">-</string>
 
-    <string name="label_github" translatable="false">GitHub</string>
-    <string name="label_translation">Translation</string>
-    <string name="label_beta">Get Beta</string>
-    <string name="label_timeline">Timeline</string>
+    <string name="label_github">Source code</string>
+    <string name="label_translation">Translations</string>
+    <string name="label_beta">Download Beta</string>
+    <string name="label_timeline">OS Timeline</string>
     <string name="label_privacy_policy">Privacy</string>
     <string name="label_license">License</string>
-    <string name="label_email">Feedback</string>
-    <string name="label_star">Star</string>
+    <string name="label_email">Send Feedback</string>
+    <string name="label_star">Rate the App</string>
 
-    <string name="label_version" translatable="false">v%s (%d)</string>
+    <string name="label_version">You're running version %s (%d)</string>
 
     <string name="label_timeline_title">Timeline, milestones, and updates</string>
     <string name="label_timeline_releases">Releases</string>
 
-    <string name="message_trypophobia_warning">This is a warning to those with &lt;b&gt;Trypophobia&lt;/b&gt; that some images may cause discomfort. Do you want to continue?</string>
+    <string name="message_trypophobia_warning">This is a trigger warning to those with &lt;b&gt;Trypophobia&lt;/b&gt; that some images may cause discomfort. Do you want to continue anyway?</string>
+
+    <string name="redesign_notice">Easter Eggs has a new look!</string>
```

Comment:
Where is this resource being used?

Reply:
Just in case the UI gets another redesign.

Reply:
I just repurchased it into an error string